### PR TITLE
add delete message

### PR DESCRIPTION
**Pull request by aloupfor**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1228

### CODE REVIEW

File: BaseBundle/Resources/translations/messages.en.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
 open_orchestra_base:
     form:
         submit: Save
+        delete: Delete
```
Comment (nicolasThal) [inline BaseBundle/Resources/translations/messages.en.yml]: eof

File: BaseBundle/Resources/translations/messages.fr.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
 open_orchestra_base:
     form:
         submit: Sauvegarder
+        delete: Supprimer
```
Comment (nicolasThal) [inline BaseBundle/Resources/translations/messages.fr.yml]: eof

**Comment by nicolasThal** [comment]
:+1: with the two eofs fixed :)

**Comment by Thiblef** [comment]
:+1:

**Pull request merged**